### ISSUE TITLE
ImportError: cannot import name 'AdaptiveFancyArrow'

### ISSUE THREAD

**Issue by rowenaliv** [biotite-dev/biotite#302]
Hey, 
I am trying to import a sub package of biotite to visualise the alignment of two protein sequences, but I receive following error: "_cannot import name 'AdaptiveFancyArrow' from 'biotite_'" in response to this code:
```python
import biotite.sequence.graphics as graphics
fig, ax = plt.subplots(figsize=(2.0,0.8))
graphics.plot_alignment_similarity_based(
    ax, alignments[0], matrix=p_matrix, symbols_per_line=len(alignments[0])
)
fig.tight_layout()
```
I use Python 3.9.2 and Biotite-0.27.0 in JupyterLab Notebook.
The exact traceback is:
```
/usr/local/Cellar/jupyterlab/3.0.8/libexec/lib/python3.9/site-packages/biotite/sequence/graphics/__init__.py
    ---> 31 from .features import *
/usr/local/Cellar/jupyterlab/3.0.8/libexec/lib/python3.9/site-packages/biotite/sequence/graphics/features.py
    ---> 14 from ...visualize import colors, AdaptiveFancyArrow
```
Thank you very much in advance :)

**Comment by padix-key** [comment]
This sounds interesting. Can you tell me which OS you use and how you installed Biotite (conda, pip)?

**Comment by rowenaliv** [comment]
I have successfully installed matplotlib previously and it works properly when using other functions.

**Comment by padix-key** [comment]
Can you tell me which OS you are using and how you installed Biotite (conda, pip)?

**Comment by rowenaliv** [comment]
I am using macOs Big Sur version 11.1 (20C69) and I installed Biotite using pip3. The matplotlib version is 3.4.0. 

When I am using the code you provided, I still receive the same traceback. However, I am not sure, if I inserted the code in the correct place. I don't know if that could be the problem. 

```python
import biotite.sequence.graphics as graphics
try:
    # Only create this class when matplotlib is installed
    from matplotlib.transforms import Bbox
    from matplotlib.patches import FancyArrow
    from matplotlib.patheffects import AbstractPathEffect

    class AdaptiveFancyArrow(FancyArrow):
        ...

except ImportError:
    pass
fig, ax = plt.subplots(figsize=(2.0,0.8))
graphics.plot_alignment_similarity_based(
    ax, alignments[0], matrix=p_matrix, symbols_per_line=len(alignments[0])
)
fig.tight_layout()
```

Thank you very much for your help. :)

**Comment by padix-key** [comment]
My code snippet was just an excerpt from the original Biotite source code in order to explain why your error might appear. You don't have to include it in your source code. 

May I also ask you to execute the following code and post the output here?
```python
import biotite.visualize as vis
print(vis.__all__)
```

**Comment by rowenaliv** [comment]
This is the output: 
```python
['colors', 'set_font_size_in_coord']
```

**Comment by padix-key** [comment]
As assumed the `AdaptiveFancyArrow` class is not created due to an `ImportError`, which points to a missing or erroneous installation of Matplotlib. The lines
```python
from matplotlib.transforms import Bbox
from matplotlib.patches import FancyArrow
from matplotlib.patheffects import AbstractPathEffect
```
did run without problems?

**Comment by rowenaliv** [comment]
Thanks for the patience with me!
Yes, they run indeed without any trouble.

**Comment by padix-key** [comment]
I am almost out of ideas. Finally, I would suggest that you remove the *try-except* block around `AdaptiveFancyArrow` in your local *Biotite* installation (`/usr/local/Cellar/jupyterlab/3.0.8/libexec/lib/python3.9/site-packages/biotite/visualize.py`, line `123` + `227` + `228`) and restart the Jupyter kernel. Maybe then there is a more clear exception raised.

**Comment by rowenaliv** [comment]
Hey @padix-key, I removed the _try-except_ block as you advised. And indeed, your former hint regarding _matplotlib_ seemed about right. Now I had an import error for _matplotlib_ whilst importing _biotite_.
In the end, I found that _matplotlib_ gets installed into `/usr/local/lib/python3.9/site-packages` and this place won't be checked at import time. Not sure, if this is a problem of Jupyter Lab.
A simple `sys.path.append(...)` mitigated this issue and everything seems to work fine now (even with the _try-except_ back into its place).
Thank you very much for your help! I really appreciate it!
